pkg/vulnsrc/glad/glad.go

```diff
@@ -35,8 +34,8 @@ var (
 		"GMS",
 	}
 
+	// Mapping between GLAD slug and Trivy ecosystem
 	ecosystems = map[packageType]types.Ecosystem{
```

Comment:
I didn't delete this map because of 2 points:
- don't forget that we need to compare the trivy-db ecosystem and the GLAD pool.
- keep the easy way to add a new ecosystem if needed.